### PR TITLE
kubeshark pattern

### PR DESCRIPTION
**Pull request by zghanem0**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/kubeshark.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Access Kubeshark
+
+Apply the kubernetes dashboard manifest.
+
+```sh
+$ kubectl -n kube-system port-forward svc/kubeshark-front 3000:80
+```
+
+
+Open the [dashboard](http://localhost:3000)
```
Comment (elamaran11) [inline docs/patterns/kubeshark.md:77]: Would you be able to show some sample network monitoring for a sample workload with some screenshots to view real usage.

**Review by elamaran11 — COMMENTED** [review]
@zghanem0 Looks very good. Could you please address this minor comment. Also the doc should be added to `mkdocs.yaml` so the website publishing automation adds the documentation to our web - https://aws-samples.github.io/cdk-eks-blueprints-patterns/

File: docs/patterns/kubeshark.md (hunk 1 above)
Comment (zghanem0) [reply docs/patterns/kubeshark.md:78]: Done!

File: docs/patterns/kubeshark.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Then you should be able to see view like this
+![dashboard](https://raw.githubusercontent.com/kubeshark/assets/master/png/kubeshark-ui.png)
+
+
+## Example
+
+In the below example, i'm writing a query to identify the traffic flow from the pod "nginx" in the "default" namespace to "aws.com" and "coredns". The query is writen by [Kubeshark Filter Language (KFL)](https://docs.kubeshark.co/en/filtering#kfl-syntax-reference) is the language implemented inside kubeshark/worker that enables the user to filter the traffic efficiently and precisely.
```
Comment (elamaran11) [inline docs/patterns/kubeshark.md:86]: How is the nginx pod deplyed, lets provide complete instructions.

**Review by elamaran11 — COMMENTED** [review]
@zghanem0 One more minor doc update required

File: docs/patterns/kubeshark.md (hunk 2 above)
Comment (zghanem0) [reply docs/patterns/kubeshark.md]: Done!.

**Review by elamaran11 — COMMENTED** [review]
@zghanem0 Im currently validating your pattern but your doc is misaligned for pattern. Please fix it.

File: docs/patterns/kubeshark.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+
+3.) Access kubeshark using the below command.
+```
+kubectl -n kube-system port-forward svc/kubeshark-front 3000:80
+```
+
+
+4.) Run Kubeshark query to identify the traffic flow.
+
```
Comment (elamaran11) [inline docs/patterns/kubeshark.md:114]: Paste the KFL query as plain text so people can copy and paste. I have to manually type and have to avoid typos.

**Review by elamaran11 — COMMENTED** [review]
@zghanem0 I have validated the pattern, works great, wonderful day 2 Ops tooling. Fix these issues, we can merge this.

**Comment by elamaran11** [comment]
@zghanem0 Please fix GH Action errors.

File: docs/patterns/kubeshark.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
+# Kubeshark AddOn
```
[the remark this answers is not in the document]
Comment (zghanem0) [reply docs/patterns/kubeshark.md:1]: Done!, I have updated the repo to meet our patterns standards

**Comment by elamaran11** [comment]
@zghanem0 Couple of more minor stuff. `mkdocs.yaml` update of your pattern is missing and add your pattern  list of patterns in `readMe.md`

**Review by elamaran11 — APPROVED** [review]
LGTM. Thankyou for the contribution.

**Comment by zghanem0** [comment]
@elamaran11 I will maintain it  and will plan to move it under aws-samples I after get approval

**Comment by elamaran11** [comment]
@zghanem0 Merging this based on your note. Please followup and update once the addon is moved to aws-samples.

**Pull request merged**